build/lib/release.sh

```diff
@@ -217,6 +218,12 @@ function kube::release::build_server_images() {
     if [[ "${KUBE_BUILD_HYPERKUBE}" =~ [yY] ]]; then
       cp "${LOCAL_OUTPUT_BINPATH}/${platform}/hyperkube" "${release_stage}/server/bin"
     fi
+    # if we are building conformance, we also need to copy the binaries needed
+    if [[ "${KUBE_BUILD_CONFORMANCE}" =~ [yY] ]]; then
+      cp "${LOCAL_OUTPUT_BINPATH}/${platform}/e2e.test" "${release_stage}/server/bin"
```

Comment:
fwiw I think we should call it kube_tests and include the other suites.  We had an issue open for a while to include the node tests to evaluate prior to running conformance.

Reply:
Ack. i want to get the basic skeleton working and hand it off to @brahmaroutu or @duglin who promised to help.

Reply:
`cp  "${LOCAL_OUTPUT_BINPATH}/${platform}/*.test" "${release_stage}/server/bin"` should work.

Reply:
Done @timothysc

Reply:
I guess this works. I'd think of this as part of building the test tarball rather than the server (=control plane) images ... Anyone building those will need to opt out of another image now or lose build time again.

Reply:
switching back to `cp "${LOCAL_OUTPUT_BINPATH}/${platform}/e2e.test" "${release_stage}/server/bin"`, will revisit later when we add node conformance test etc

Reply:
Why do you need these test binaries in the server tarball? I don't see them used anywhere.

Reply:
Ideally, we would want e2e.test and e2e_node.test - When a group is running conformance this gives them the ability to check to see it the node itself passes the local checks.